src/main/java/codeu/model/data/Conversation.java

```diff
@@ -25,7 +25,8 @@
   public final UUID id;
   public final UUID owner;
   public final Instant creation;
-  public final String title;
+    public String title;
```

Comment:
Actually I dont need to be able to change the conversation name; will make this final again and remove the setter

- [ ] Return final modifier and remove setter